site/content/docs/0.0/layout/breakpoints.md

```diff
@@ -11,4 +11,188 @@ aliases:
 toc: true
 ---
 
-{{< callout-soon "page" >}}
+## Core concepts
+
+- **Breakpoints are the building blocks of responsive design.** Use them to control when your layout can be adapted at a particular viewport or device size.
+
+- **Use media queries to architect your CSS by breakpoint.** Media queries are a feature of CSS that allow you to conditionally apply styles based on a set of browser and operating system parameters. We most commonly use `min-width` in our media queries.
+
+- **Mobile first, responsive design is the goal.** OUDS Web's CSS aims to apply the bare minimum of styles to make a layout work at the smallest breakpoint, and then layers on styles to adjust that design for larger devices. This optimizes your CSS, improves rendering time, and provides a great experience for your visitors.
+
+## Available breakpoints
+
+OUDS Web includes eight default breakpoints, sometimes referred to as _grid tiers_, for building responsively. These breakpoints can be customized if you're using our source Sass files.
+
+{{< bs-table "table" >}}
+| Breakpoint | Class infix | Dimensions |
+| --- | --- | --- |
+| 2X-small | <em>None</em> |&lt;390px |
```

Comment:
We should probably use the name from https://unified-design-system.orange.com/472794e18/p/26a48e-grid/t/e076cd8936 or Bootstrap

Reply:
Not sure, because we would have almost exactly the same content in the first and second column? This way seems a little bit more explicit maybe?